packages/devtools_app/lib/src/screens/memory/memory_events_pane.dart

```diff
@@ -157,7 +156,7 @@ class MemoryEventsPaneState extends State<MemoryEventsPane>
   EventChartController get _chartController => widget.chartController;
 
   /// Controller for managing memory collection.
-  MemoryController _memoryController;
+  late MemoryController _memoryController;
```

Comment:
use `initialize` pattern we've been using elsewhere

Reply:
done